client/app/bundles/course/lesson_plan/items.jsx

```diff
@@ -1,5 +1,5 @@
 import initializeComponent from 'lib/helpers/initializeComponent';
 import storeCreator from './store';
-import LessonPlanContainer from './containers/LessonPlanContainer';
+import LessonPlan from './containers/LessonPlan';
 
-initializeComponent(LessonPlanContainer, 'lesson-plan-items', storeCreator);
+initializeComponent(LessonPlan, 'lesson-plan-items', storeCreator);
```

Comment:
Sorry I didn't catch this in your last PR. 

I think it might not be good to make `initializeComponent` a standard helper, most of the component won't use a second request to get the JSON data. it's a bit over abstracted to me also... Having a `ProviderWrapper` is good enough to me.  What you think ?